ern-container-gen-ios/src/hull/ElectrodeContainer/ElectrodeReactNative.h

```diff
@@ -50,6 +50,10 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, copy) NSString *packagerPort;
 @end
 
+@protocol MiniAppViewDelegate <NSObject>
+- (void)rootViewDidChangeIntrinsicSize:(UIView *)rootView;
```

Comment:
Who will trigger this method call? Is it supposed to be invoked after this [line](https://github.com/electrode-io/electrode-native/pull/1332/files#diff-7cae239e661f25961b5b8e45e627f218R167)  ?

Reply:
this is getting called when rootView's content size updated to a new size @deepueg